### ISSUE TITLE
News-Categories Import wird nicht angezeigt

### ISSUE THREAD

**Issue by Shoekrates** [inspiredminds/contao-wordpressimport#5]
Hi!
Der Import ohne Kategorien läuft einwandfrei, das ist toll. Leider klappt das Zusammenspiel mit codefog/contao-news_categories nicht so richtig – die Kategorie-Option wird mir nicht angezeigt.

- Contao 4.9.5
- Wordpress-Importer 1.1.2
- codefog/contao-news_categories 3.3.1

Ich habe testweise eine Kategorie angelegt, kann diese auch im News-Archiv weiter oben als Einschränkung auswählen, aber bei den Importer-Optionen tauchen die Kategorien nicht auf.

Hat sich da vielleicht bei den Updates beider Erweiterungen in der Zwischenzeit etwas geändert, so dass der Importer die Categories-Erweiterung nicht mehr erkennt?

**Comment by fritzmg** [comment]
Ja, diese Extension wurde noch nur für Version 2.x der `codefog/contao-news_categories` Erweiterung geschrieben. Für Version 3.x müsste die Erweiterung aktualisiert werden.

Aktuell haben wir leider keine Projekte, wo das gebraucht wird, daher kann ich nicht sagen, wann wir das implementieren können. Ansonsten hilft mir aber Sponsoring dabei 😉  (siehe rechts oben) - oder Beauftragungen sind natürlich auch möglich (oder jemand anders beauftragen einen PR dafür zu machen geht natürlich auch, ist ja Open Source). Grundsätzlich sind die notwendigen Anpassungen vermutlich sehr gering.

**Comment by Shoekrates** [comment]
OK, danke für die Info. Momentan geht es um die prinzipielle Machbarkeit für ein größeres laufendes Wordpress-Projekt. Es juckt halt in den Fingern, das Projekt auf Contao umzustellen ;-)
Wir denken das mal weiter durch und ich komme gerne auf das Sponsoring zurück, wenn wir eine Entscheidung getroffen haben.

Ich nehme an, dass eine Eingrenzung des Imports auf bestehende Wordpress-Kategorien auch ein Feature-Request wäre, oder?
Also z. B. Wordpress-Kategorie X wird importiert in Contao-News-Archiv X, WP-Kategorie Y wird importiert in Contao-News-Archiv Y.

**Comment by fritzmg** [comment]
> Ich nehme an, dass eine Eingrenzung des Imports auf bestehende Wordpress-Kategorien auch ein Feature-Request wäre, oder?
> Also z. B. Wordpress-Kategorie X wird importiert in Contao-News-Archiv X, WP-Kategorie Y wird importiert in Contao-News-Archiv Y.

Ja, das wäre auf jeden Fall ein neues Feature. Aber vielleicht nicht unbedingt notwendig - man kann ja erst mal alles so wie es ist importieren lassen und danach nach Kategorien filtern und die Nachrichtenbeiträge in andere Nachrichtenarchive verschieben.

**Comment by Shoekrates** [comment]
Stimmt, das setzt dann aber ein funktionierendes Zusammenspiel mit der Categories-Erweiterung voraus.
Danke schon mal für die flotten Antworten!

**Comment by fritzmg** [comment]
Support for newer versions was added in [`1.4.0`](https://github.com/inspiredminds/contao-wordpressimport/commit/adf8115db40fdcca78a28592996b9e1310271398).
